### ISSUE TITLE
Fixing CI's conflicting behaviour

### ISSUE THREAD

**Issue by stealthanthrax** [fossasia/susi_desktop#116]
**Actual Behaviour**

Travis asks for tabs in indentation and codacy asks for spaces

**Expected Behaviour**

To chose either one

**Steps to reproduce it**

Make a PR

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.
